package/src/components/Message/utils/messageActions.ts

```diff
@@ -43,33 +43,33 @@ export const messageActions = <
   threadReply,
   unpinMessage,
 }: {
-  blockUser: MessageAction | null;
+  blockUser: MessageActionListItemMainProps | null;
```

Comment:
Could we maybe change these so we can let them be omitted to define them as `null`, or is there a reason we make it required to explicitly set the value of the prop as null? Do you know/remember why it was done like this originally, @vishalnarkhede?
```
  blockUser?: MessageActionListItemMainProps;
```